activation/poet.go

```diff
@@ -492,7 +500,12 @@ func (c *poetService) authorize(
 	cert, err := c.Certify(ctx, nodeID)
 	switch {
 	case err == nil:
-		return &PoetAuth{PoetCert: cert}, nil
+		info, err := c.getInfo(ctx)
+		if err != nil {
+			return nil, err
+		}
+
+		return &PoetAuth{PoetCert: cert, CertPubKey: info.Certifier.Pubkey}, nil
```

Comment:
It should continue to work the way it was (without passing the hint).

Reply:
I don't understand? Sure the node just can't provide the certifier pubkey hint and then everything works like before, but if the nodes certificate has been signed by a different Certifier than the one returned from `/info` of the PoET it is using then it should be able to at least try to still submit in case the PoET accepts certificates from the certifier the node is using or not?